ci-operator/config/crc-org/crc/crc-org-crc-main.yaml

```diff
@@ -43,15 +43,15 @@ resources:
 tests:
 - as: e2e-crc
   steps:
-    cluster_profile: gcp-3
+    cluster_profile: osl-gcp
     workflow: code-ready-crc-e2e
 - as: integration-crc
   steps:
-    cluster_profile: gcp-3
+    cluster_profile: osl-gcp
     workflow: code-ready-crc-integration
 - as: e2e-microshift-crc
   steps:
-    cluster_profile: gcp-3
+    cluster_profile: osl-gcp
```

Comment:
`osl` stands for OpenShift Local?
Please, nly use this for technical reasons, as this is not an official abbreviation.

Reply:
yes, it is not offical abbreviation but need to use a short form for technical reason.

Reply:
Why not `crc`?